Gopkg.toml

```diff
@@ -160,8 +160,8 @@
   version = "~v2.2.2"
 
 [[constraint]]
-  name = "gopkg.in/zorkian/go-datadog-api.v2"
-  version = "~v2.12"
+  name = "github.com/CharlyF/go-datadog-api"
```

Comment:
temporary until we merge upstream

Reply:
you should use `[[override]]` like this you don't need to update the `import gopkg.in/zorkian/go-datadog-api.v2` in the golang files

Reply:
yeah, it's really temporary I will remove it overall when we merge in zorkian. But good point.

Reply:
I'm not against the usage of a fork here. it just a pro-tips to avoid change like: https://github.com/DataDog/datadog-agent/pull/4669/files#diff-c40534601562c617a398940e61fe356dR16